src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -52,17 +52,40 @@ import scala.collection.immutable.ListMap
     |
     |## Available Filters
     |
+    |### A-tailing Artifact Filter
+    |
+    |The A-tailing artifact filter attempts to measure the probability that a SNV is the product of
+    |errors in the template generated during the A-base addition steps that are common to many Illumina
+    |library preparation protocols.  The artifacts occur if/when a recessed 3' end is incorrectly filled
+    |in with one or more adenines during A-base addition. This presents specifically as errors to T at
```

Comment:
Avoid using ambiguous pronouns in docs as it may not be clear what the subject is

```suggestion
    |in with one or more adenines during A-base addition. Incorrect adenine incorporation presents specifically as errors to T at
```